.github/workflows/herokuCi.yml

```
@@ -0,0 +1,37 @@
+# This is a basic workflow to help you get started with Actions
+
+name: Heroku CI
+
+# Controls when the workflow will run
+on:
+  # Triggers the workflow on push or pull request events but only for the master branch
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
```

```suggestion

  # Allows you to run this workflow manually from the Actions tab
  workflow_dispatch: ~
```
No need to deploy on PR